docker-compose.yml

```diff
@@ -95,23 +93,17 @@ services:
     image: dlrriesgos/frontend
     depends_on:
       - backend
-    ports:
-      - 8000:80
     environment:
-      subPath: ${frontendSubPath}
```

Comment:
Should we remove this one from the .env file too? (And from the readme.md?)

Reply:
Good point, oversight! 
`frontendSubPath` and `frontendLightSubPath` removed from .env.